database/cassandraSchema.cql

```diff
@@ -0,0 +1,27 @@
+DROP KEYSPACE IF EXISTS dgdb;
+CREATE KEYSPACE dgdb WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'}  AND durable_writes = true;
+
+CREATE TABLE dgdb.listing (
+  ID INT PRIMARY KEY,
+  perNight INT ,
+  Rating DECIMAL,
+  RatingAmount INT,
+  guestsAllowed INT,
+  guestsInfants INT,
+  cleaningFee DECIMAl,
```

Comment:
You have a few DECIMAl typos in here that I would change

Reply:
Good catch! I'll fix it before I submit.